recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,210 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class LibITKConan(ConanFile):
+    name = "itk"
+    homepage = "http://www.itk.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    description = "Insight Segmentation and Registration Toolkit"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_jpegturbo": [True, False]}
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "use_jpegturbo": False}
+    generators = "cmake", "cmake_find_package"
+    short_paths = True
+    _source_subfolder = "sf"
+    _build_subfolder = "build"
+    _upstream_version= ""
+    _upstream_patch= ""
+
+    # TODO: Some packages can be added as optional, but they are not in CCI:
+    # - mkl
+    # - fftw
+    # - vtk
+    # - opencv
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
Do not get the _sources_ in the configure method, you are already doing it in the `source` one

Reply:
done

Reply:
done